web/src/app/settings/single-setting/single-setting.component.ts

```diff
@@ -21,16 +20,18 @@ export class SingleSettingComponent implements OnInit {
     @Output() update = new EventEmitter();
 
     private schedules: Schedule[];
+    private settingInEditMode: boolean;
```

Comment:
_setting_**In**_EditMode_ sounds like statement. I think _settingEditMode_ is better to use here instead. This term is used in settings state holder functions, as well